### PR TITLE
Update peptides_proteins.rst

### PR DESCRIPTION
**Pull request by timosachsenberg**

### CODE REVIEW

**Comment by timosachsenberg** [comment]
let's  wait for CI and see what was wrong before the merge and revert...

**Comment by jpfeuffer** [comment]
Please check the old PR for my review.

**Comment by timosachsenberg** [comment]
Hmm isn’t that this PR? Don’t see them on my phone

**Comment by jpfeuffer** [comment]
https://github.com/OpenMS/pyopenms-docs/pull/391

**Comment by timosachsenberg** [comment]
@Ayesha-Feroz can you check Julianus comments and try to address them?

File: docs/source/user_guide/peptides_proteins.rst
Comment (Ayesha-Feroz) [inline docs/source/user_guide/peptides_proteins.rst:342]: sequence = poms.AASequence.fromString("TESTMTECSTMTESTR")

**Review by Ayesha-Feroz — CHANGES_REQUESTED** [review]

File: docs/source/user_guide/peptides_proteins.rst
Comment (timosachsenberg) [reply docs/source/user_guide/peptides_proteins.rst:338]: yes please change and test it in binder.
e.g., this change would be an error I guess because indentation is not correct.

File: docs/source/user_guide/peptides_proteins.rst
Comment (timosachsenberg) [reply docs/source/user_guide/peptides_proteins.rst:342]: yep I think you can change it like that

File: docs/source/user_guide/peptides_proteins.rst
Comment (timosachsenberg) [reply docs/source/user_guide/peptides_proteins.rst:347]: exactly, this makes it clear that a static method is called

File: docs/source/user_guide/peptides_proteins.rst
Comment (timosachsenberg) [reply docs/source/user_guide/peptides_proteins.rst:350]: correct

File: docs/source/user_guide/peptides_proteins.rst
Comment (timosachsenberg) [reply docs/source/user_guide/peptides_proteins.rst:354]: correct

File: docs/source/user_guide/peptides_proteins.rst
Comment (timosachsenberg) [reply docs/source/user_guide/peptides_proteins.rst:362]: yes but I think the last "," is to much / not needed. can you confirm?

**Pull request merged**
